lib/projection_manager.ex

```diff
@@ -235,6 +236,18 @@ defmodule ProjectionManager do
       )
     end
 
+    :telemetry.execute(
+      [:projection_manager, :refresh],
+      %{
+        duration: System.monotonic_time() - start_time
+      },
+      %{
+        projector: projector,
+        projection_ids: projection_ids,
```

Comment:
Metadata fields can only have a value of an Atom or a Binary. I'm reasonably confident most telemetry systems (like our own influxdb) will reject these points.